pkgs/applications/science/logic/cadical/default.nix

```
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "cadical";
+  version = "1.2.1";
+
+  src = fetchFromGitHub {
+    owner = "arminbiere";
+    repo = pname;
+    rev = "92d72896c49b30ad2d50c8e1061ca0681cd23e60";
+    hash = "sha256:1a66xkw42ad330fvw8i0sawrmg913m8wrq5c85lw5qandkwvxdi6";
+  };
+
+  configurePhase = "./configure";
+  # nativeBuildInputs = [ make ];
+  buildInputs = [ ];
```

```suggestion
  dontAddPrefix = true;
```
It seems this was necessary because the configure script does not support `--prefix`. It's helpful for review and future maintenance to have information like this in comments. Anyways, the generic configure phase does cover this special case (see [here](https://github.com/NixOS/nixpkgs/blob/ec9cf578e56ff7c3045878f86506338b0112777d/pkgs/stdenv/generic/setup.sh#L970)), so we can just use that :+1:

---

I didn't know the attribute.
OK, I've updated as you suggested. Any other problem? 😅

---

I didn't precisely know the attribute either :) I just thought "Huh, `configurePhase = "./configure";` looks like it should be default, what's going on there?" and investigated a little :)

Now you can remove lines 14-16, they don't do anything anymore.

Thank you for creating the package! :+1:

---

OK, I've just removed these useless lines. Thank you too very much! I learned much.